debian-buster/symbiflow-eos-s3.dockerfile

```diff
@@ -0,0 +1,55 @@
+# Authors:
```

Comment:
Please, merge this dockerfile with `symbiflow-xc7,dockerfile`, name it `symbiflow.dockerfile`, and use stages for avoiding duplication. Note that not all stages need to be executed. Moreover, the final images don't need to be built at once. See nextpnr or sim for different strategies in this regard.